Quasardb/QdbTimeSeries.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Quasardb.Native;
+using Quasardb.TimeSeries;
+using System;
+
+namespace Quasardb.TimeSeries
+{
+    /// <summary>
+    /// A timeseries
+    /// </summary>
+    [Obsolete("Use Quasardb.QdbTable instead")]
+    public class QdbTimeSeries : QdbTable
```

Comment:
What's the rationale behind having this? This change breaks everything anyway, nope?

Reply:
No, everything else is expected to continue working, and it should to the best of my .net knowledge (which isn't much to begin with). This is without considering not-yet-released interfaces like the new reader and writer. I'd appreciate a thorough review on the backwards compatibility aspect.

This class `QdbTimeSeries` and `QdbCluster.TimeSeries()` are the two pieces needed for backwards compatibility.

Reply:
Roger that. Have you tested it for backwards compatibility by reverting all changes in tests?

Reply:
That's exactly how I tested it! But it may not be sufficient, I'm thinking for instance that `QdbTimeSeries` is never mentioned explicitly but always inferred.

Reply:
OK. I must admit that I don't know how to test it better then.
Maybe someone else knows.
BTW, is there an equivalent of `using/typedef` in C# so that you just write `using QdbTimeSeries = QdbTable;`? :)
Otherwise, LGTM.

Reply:
It has a `using`, but as far as I could find an alias is not something that can be "exported" from a namespace, it only exists within the file.